core/src/main/java/com/netease/arctic/trace/TableTracer.java

```diff
@@ -53,13 +54,25 @@ public interface TableTracer {
   void deleteDeleteFile(DeleteFile deleteFile);
 
   /**
-   * Commit table changes.
+   * Replace some properties of table
+   * @param newProperties properties to replace
    */
-  void commit();
+  void replaceProperties(Map<String, String> newProperties);
 
   /**
    * Replace some properties of table
-   * @param newProperties properties to replace
+   * @param updateColumn updated column info
    */
-  void replaceProperties(Map<String, String> newProperties);
+  void updateColumn(AmsTableTracer.UpdateColumn updateColumn);
```

Comment:
Interface should not depend on a inner class of its implementation class

Reply:
will change in new commit